services/git-operator/0.1.0/git-operator-manifests/all.yaml

```diff
@@ -955,7 +955,7 @@ spec:
         app.kubernetes.io/name: git-operator
     spec:
       containers:
-      - image: ghcr.io/mesosphere/dkp-container-images/docker.io/nginxinc/nginx-unprivileged:1.25.5-alpine-d2iq.0
+      - image: docker.io/nginxinc/nginx-unprivileged:1.27.1-alpine
```

Comment:
I think this change needs to be reverted in this pr. this will be part of git-operator bump PR.